scripts/cli.py

```diff
@@ -189,7 +189,7 @@ def check_env(self):
         logging.info('Using KNoT base path: ' + self.knot_path)
 
         # Get current working directory
-        self.cwd = os.getcwd()
+        self.cwd = os.getcwd().replace(" ","\ ")
```

Comment:
About commit message:
**R**eplacing

Reply:
What are you fixing? What is CWD? 
Explain what was the problem. The commit message should explain at least what is the problem and what solves it.

Reply:
spaces with "\ " **character**

Reply:
Please use the function https://docs.python.org/2/library/pipes.html#pipes.quote for python 2
or https://docs.python.org/3.4/library/shlex.html#shlex.quote for python 3